### ISSUE TITLE
[Bug]Message text in MMO not work

### ISSUE THREAD

**Issue by moepi2k** [suriyun-production/mmorpg-kit-docs#1346]
https://gyazo.com/fbcb05a044f58e195af6775bb1f08557

like u can see in video you always just get "Message"
also when you invite someone for guild/party. and other decline it
you should need to get a msg for decline.

-decline msg only works for trading

**Comment by insthync** [comment]
You can fix it by add codes to `UIGameMessageHandler.cs`

at line 62 
```
            newMessage.text = LanguageManager.GetText(message.ToString());
```

**Comment by moepi2k** [comment]
great thx :)
now i see that the decline msg is there, but on the wrong client :)
https://gyazo.com/ac3001f2301b308e4142b3957137c800

**Comment by insthync** [comment]
Fixed in 1.72c4
